publications.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Publications
+---
+
+
+<div class="colorlib-intro colorlib-bg-white">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-md-offset-1 colorlib-heading animate-box">
```

Comment:
Since you have `animate-box` for each list item, you don't need it for the entire `div`. Put it for the `h2` tag below though.